### DISCUSSION TITLE
Plugins menus displayed even before login

### DISCUSSION THREAD

**Discussion by m-roshani** [mattieserver/netbox-topology-views#356] [Q&A] [answered]
Hi,
Environment: 
Ubuntu Focal 20.04
Netbox-3.4.1
Topology plugin: 3.5.0

We are using Netbox LDAP authentication and now even before login the plugin menus are in the home page.
By the way, clicking on them redirects again to login. As my colleague found out, seems there are no permissions set for menu items here: 

https://github.com/mattieserver/netbox-topology-views/blob/develop/netbox_topology_views/navigation.py#L40-L48
`permissions` parameter for `PluginMenuItem` as stated in the [docs]( https://docs.netbox.dev/en/stable/plugins/development/navigation/#menu-items) seems missing. 

Is that the reason or we can fix it in some other way?

Thanks

**Comment by mattieserver** [comment, marked as answer]
this is now fixed in [v3.6.2](https://github.com/mattieserver/netbox-topology-views/releases/tag/v3.6.2)
